### DISCUSSION TITLE
Webview for WebGL

### DISCUSSION THREAD

**Discussion by FrankDaze** [nhn/gpm.unity#347] [Q&A]
Is it planned to make a version for WebGL as well?
Or is it maybe already working with WebGL?

Thanks in advance 
Frank

**Comment by SangYun-nhn** [comment]
Dooray! 부재중 자동 응답










부재중 자동 응답






***@***.***) 

님은 현재 부재중입니다.


2023.01.19 ~ 2023.01.27


(이 메일은 부재중 
설정을
통해 자동으로 전송되는 메일입니다.)






[부재기간 : 2023-01-19 09:30 ~ 2023-01-27 18:30 GMT +09:00]휴가 중입니다.급한 용무가 있으시면 휴대폰으로 연락바랍니다.






이 메일은 발신전용으로 회신되지 않습니다.





© Dooray!

**Comment by FrankDaze** [comment]
Sorry it's not possible for me to translate your answer.

**Reply by SangYun-nhn** [reply]
@taehyeon-nhn 

Please check this comment

**Comment by taehyeon-nhn** [comment]
Hello. @FrankDaze 

There are no plans to support WebGL yet.
We will discuss the contents within the team.

Thank you.

**Comment by FrankDaze** [comment]
thanks for your answer :)

**Comment by taehyeon-nhn** [comment]
Hello. @FrankDaze.

Could you tell me what WebGL WebView is needed for?

Thank you.

**Reply by FrankDaze** [reply]
Sure, I plan to make a something like "Workadventure". A top down pixel graphic game, where you can run through an office and if you enter a conference room then a webview opens with contains a video conference like Jitsi. 

That was my plan, but now I found our, that Unity3D still can't access the microphone from WebGL. So, this will not work.

**Reply by taehyeon-nhn** [reply]
Thank you for your quick response. It is very helpful.
